visualization/app/codeCharta/ui/codeMap/codeMap.component.spec.ts

```diff
@@ -85,6 +98,29 @@ describe("ColorSettingsPanelController", () => {
 		})
 	})
 
+	describe("onSharpnessModeChanged", () => {
+		const sharpnessTests = [
+			{ func: "stopAnimate" },
+			{ func: "destroy" },
+			{ func: "autoFitTo" },
+			{ func: "animate" },
+			{ func: "animateStats" }
+		]
+		beforeEach(() => {
+			codeMapController.onSharpnessModeChanged()
+		})
+
+		for (const test of sharpnessTests) {
+			it(`should call threeViewerService ${test.func}`, () => {
+				expect(threeViewerService[`${test.func}`]).toHaveBeenCalled()
+			})
+		}
```

Comment:
I personally would have used a single `it` and used a for loop inside that, but that's just my personal style and not blocking.